contracts/contracts.go

```diff
@@ -31,6 +38,35 @@ func GetLoadTesterBytes() ([]byte, error) {
 	return hex.DecodeString(RawLoadTesterBin)
 }
 
+func BlockUntilSuccessful(ctx context.Context, c *ethclient.Client, retryable func() error) error {
+	// this function use to be very complicated (and not work). I'm dumbing this down to a basic time based retryable which should work 99% of the time
```

Comment:
lol looks famliar

Reply:
yeah lol it was copypasta from your previous version